websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+	clientId: "1031730262092042321",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
```

Comment:
Use Imgur links over Discord asset names.
```suggestion
		largeImageKey: "https://i.imgur.com/4mDvKNa.png",
```